components/expenses/TransactionsExportPopoverAndButton.js

```diff
@@ -77,7 +77,7 @@ class ExportForm extends React.Component {
     super(props);
 
     // Calculate the start: first day of previous month & end date: today.
-    const oneMonthAgo = moment().subtract(1, 'months')._d;
+    const oneMonthAgo = dayjs().subtract(1, 'month').$d;
```

Comment:
`toDate` would be clearer

```suggestion
    const oneMonthAgo = dayjs().subtract(1, 'month').toDate();
```